circleci-provision-scripts/base.sh

```diff
@@ -45,9 +45,12 @@ EOF
     export DEBIAN_FRONTEND=noninteractive
 
     apt-get update -y
+    apt-get install -y software-properties-common
+    apt-get update -y
+    apt-add-repository ppa:git-core/ppa
 
     # Install base packages
-    cat << EOS | xargs apt-get install
+    apt-get install $(tr "\n" " " <<EOS
```

Comment:
There should be no difference in using double quotes vs single quotes here, so it is probably safe to use double quotes everywhere. Here :point_up: